src/main/java/com/rhombus/api/examples/DoorReport.java

```diff
@@ -0,0 +1,211 @@
+package com.rhombus.api.examples;
+
+import com.rhombus.sdk.LocationWebserviceApi;
+import com.rhombus.sdk.DoorWebserviceApi;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.rhombus.ApiClient;
+import com.rhombus.sdk.domain.*;
+import org.apache.commons.cli.*;
+import org.apache.http.Header;
+import org.apache.http.message.BasicHeader;
+import org.glassfish.jersey.jackson.internal.jackson.jaxrs.json.JacksonJaxbJsonProvider;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLSession;
+import javax.ws.rs.client.ClientBuilder;
+import java.io.File;
+import java.io.FileWriter;
+import java.sql.Timestamp;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.ZonedDateTime;
+import java.util.*;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DoorReport {
+    private static ApiClient _apiClient;
+    private static DoorWebserviceApi _doorWebservice;
+    private static LocationWebserviceApi _locationWebservice;
+
+    public static void main(String[] args) throws Exception {
+        final Options options = new Options();
+        // command line arguments for the user
+        options.addRequiredOption("a", "apikey", true, "API key");
+        options.addRequiredOption("n", "sensorName", true, "Name of the sensor");
+        options.addOption("s", "startTime", true, "Start time of data collection yyyy-mm-dd 00:00:00");
+        options.addOption("e", "endTime", true, "End time of data collection yyyy-mm-dd 00:00:00");
+        options.addOption("c", "csv", true, "Path to the csv and name the csv");
+
+        // specifies what is displayed if help command given
+        final CommandLine commandLine;
+        try {
+            commandLine = new DefaultParser().parse(options, args);
+        } catch (ParseException e) {
+            System.err.println(e.getMessage());
+
+            new HelpFormatter().printHelp(
+                    "java -cp rhombus-api-examples-all.jar com.rhombus.api.examples.DoorReport",
+                    options);
+            return;
+        }
+
+        String apiKey = commandLine.getOptionValue("apikey");
+        String sensorName = commandLine.getOptionValue("sensorName");
+        String startTime = commandLine.getOptionValue("startTime");
+        String endTime = commandLine.getOptionValue("endTime");
+
+        String csv;
+        String real_csv;
```

Comment:
try to stick with a single naming convention.  In Java we use the naming convention `realCsv`